### ISSUE TITLE
hyper parameters

### ISSUE THREAD

**Issue by kilianFatras** [hongxin001/JoCoR#3]
Hello,

Could you please provide the hyper-parameters ? It would be easier to reproduce your results, especially the lambda for clothing 1M/ Cifar10/ Cifar100.

Thank you very much.

**Comment by hongxin001** [comment]
Actually, it depends on the datasets, noisy rate and basic models. Generally, the best lambda is between 0.5-0.9;

For example, in my experiment, the best lambda for CIFAR10 with 20%,50% and asym 40% noise is 0.9. When it comes to 80% noise, the best is 0.65.

If you change the basic model to ResNet or add normalization in dataloader, you need to try different lambda for the best.

**Comment by hongxin001** [comment]
For other hyper-parameters, it is fixed as described in the paper.

**Comment by hongxin001** [comment]
For Cifar100 in my experiment, the best lambda should be 0.85.

**Comment by kilianFatras** [comment]
Thank you for your answer ! Could you provide the one you used for clothing1M please ?

May I suggest you to write a small paragraph about it in your readme ? That would be simpler for many people reusing your code :).

**Comment by hongxin001** [comment]
Thanks for your advice. I will add the paragraph in readme. For Clothing1m dataset, I didn't keep the training records in my server. So I suggest you search for it between 0.5-0.9.
